backend/src/locales/en.json

```diff
@@ -1,15 +1,31 @@
 {
-	"emails": {
-		"accountMultiRegistration": {
+  "emails": {
+    "addedContributionMessage": {
+      "commonGoodContributionMessage": "you have received a message from {senderFirstName} {senderLastName} regarding your common good contribution “{contributionMemo}”.",
+      "linkToYourAccount": "Link to your account:",
```

Comment:
its likely this has to be moved on layer higher

Reply:
You mean to remove `emails`?

If we start to tranlslate errors I would put them into an `errors` entry parallel to `emails`.
Therefore I would leave it this way and it is clear what this tranlsations are for.

We alrady can start to translate errors …